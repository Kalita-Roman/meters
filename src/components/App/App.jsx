import React from 'react';
import './App.scss';

import Grid from '../Grid';

export default () => (<div className="app" >
    <p>Hello, World!</p>
    <p>Hello, World!</p>
    <Grid name="A" items={[{name: 'a'}, {name: 'b'}, {name: 'c'}]}/>
    <Grid name="B" items={[{name: 'e'}, {name: 'f'}]}/>
    <Grid name="C" items={[{name: 'g'}]}/>
</div>);
