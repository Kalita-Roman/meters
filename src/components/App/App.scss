.app {
    color: black;
    display: inline-flex;
    flex-direction: column;
}