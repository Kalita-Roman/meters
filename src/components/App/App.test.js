import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from './App';

test('App.jsx', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<App />);
    expect(result).toMatchSnapshot()
});