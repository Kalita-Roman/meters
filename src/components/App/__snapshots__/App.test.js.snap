// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.jsx 1`] = `
<div
  className="app"
>
  <p>
    Hello, World!
  </p>
  <_default
    items={
      Array [
        Object {
          "name": "a",
        },
        Object {
          "name": "b",
        },
        Object {
          "name": "c",
        },
      ]
    }
    name="A"
  />
  <_default
    items={
      Array [
        Object {
          "name": "e",
        },
        Object {
          "name": "f",
        },
      ]
    }
    name="B"
  />
</div>
`;
